.agent/skills/ss14-physics-system-api/SKILL.md

```
@@ -0,0 +1,447 @@
+1. First select the task type:
+- strength/speed,
+- body type/sleep/conflict,
+- fixtures/shapes,
+- contacts/raycasts/distances,
+- transform transformations.
+
+2. Then select the level:
+- safe gameplay method,
+- or a low-level engine method (rarely needed).
+
+3. If fixtures/collisions have changed:
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)